chart/templates/db-migrations-job.yaml

```diff
@@ -33,10 +33,6 @@ spec:
     spec:
       restartPolicy: Never
       serviceAccountName: db-migrations
-      imagePullSecrets:
```

Comment:
Ah, this won't work for all clients. Rather, we should have an `{{ if }}{{- end }}` checking if `imagePullSecrets` is set at all.

Reply:
OK

Reply:
I made it optional. But I do not understand why it should be necessary.

Reply:
just discussed in a call: It's necessary when the images come from a private registry and the node's default service account does not already have access to that registry.

Reply:
> just discussed in a call: It's necessary when the images come from a private registry and the node's default service account does not already have access to that registry.

OK, we have to check the other deployments, sts, etc. as well if it is really necessary.